CHANGELOG.md

```diff
@@ -1,16 +1,23 @@
 # UNRELEASED
 
+# matrix-sdk-crypto-wasm v14.0.0
+
 -   Fix a problem, introduced in v12.0.0, when importing the published package as an ESM module, in which some files could be incorrectly interpreted as CommonJS, leading to syntax errors.
     ([#189](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/189))
 
 -   `CollectStrategy.deviceBasedStrategy` is deprecated, and replaced by other methods in `CollectStrategy`.
     ([#194](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/194))
 
--   Update matrix-rust-sdk to `8d530ef2`, which includes:
+-   Better generated typescript types for async functions (`Promise<T>` instead of `Promise<any>`)
+-   Update matrix-rust-sdk to `v0.10.0`, which includes:
 
+    -   [breaking] CollectStrategy::DeviceBasedStrategy is now split into three separate strategies (AllDevices, ErrorOnVerifiedUserProblem, OnlyTrustedDevices), to make the behaviour clearer.
+        ([#4581](https://github.com/matrix-org/matrix-rust-sdk/pull/4581))
```

Comment:
note that this is not a breaking change as far as the wasm bindings go.

This release should be v13.1.0. I'm going to close this PR and create a new one.

Reply:
oh, but some of the typescript changes *are* breaking changes.